lddwrap/__init__.py

```diff
@@ -87,32 +88,8 @@ def as_mapping(self) -> Mapping[str, Any]:
                                         ("unused", self.unused)])
 
 
-_MEM_ADDRESS_RE = re.compile(r'^\s*\(([^)]*)\)\s*$')
-
-
-def _strip_mem_address(text: str) -> str:
-    r"""
-    Strip the space and brackets from the mem address in the output.
-
-    :param text: to be stripped
-    :return: bare mem address
-
-    >>> _strip_mem_address('(0x00007f9a1a329000)')
-    '0x00007f9a1a329000'
-
-    >>> _strip_mem_address(' (0x00007f9a1a329000) ')
-    '0x00007f9a1a329000'
-
-    >>> _strip_mem_address('\t(0x00007f9a1a329000)\t')
-    '0x00007f9a1a329000'
-    """
-    mtch = _MEM_ADDRESS_RE.match(text)
-    if not mtch:
-        raise RuntimeError(("Unexpected mem address. Expected to match {}, "
-                            "but got: {!r}").format(_MEM_ADDRESS_RE.pattern,
-                                                    text))
-
-    return mtch.group(1)
+_LDD_ARROW_OUTPUT_RE = re.compile(r"(?P<soname>.+) => (?P<dep_path>.*) \(?(?P<mem_address>\w*)\)?")
```

Comment:
I think the spaces need to be replaced with `\s` to include the tabs as well?

Reply:
Sure. I actually went with spaces to be more strict, but I can see that this is not always the case.